tests/jvm/src/test/resources/metacp.expect

```diff
@@ -503,6 +503,16 @@ com.javacp.Test#typeParams(ArrayList,Hashtable).(b) => param b: Hashtable[String
   Hashtable => java.util.Hashtable#
   String => java.lang.String#
   B => com.javacp.Test#[B]
+com.javacp.Test#vararg(Int,Array). => private[javacp] method vararg: (a: Int, args: String*): Unit
+a => com.javacp.Test#vararg(Int,Array).(a)
+Int => scala.Int#
+args => com.javacp.Test#vararg(Int,Array).(args)
+String => java.lang.String#
+Unit => scala.Unit#
+com.javacp.Test#vararg(Int,Array).(a) => param a: Int
+Int => scala.Int#
+com.javacp.Test#vararg(Int,Array).(args) => param args: String*
+String => java.lang.String#
```

Comment:
If one compares these `foo => bar` lines with other lines nearby, they look misaligned. @olafurpg and I typically use `sbt save-expect` to update expect files automatically to avoid this kind of mishaps.

Reply:
nice trick, the latest commit should fix it